src/main/resources/assets/mo/lang/lol_US.lang

```diff
@@ -0,0 +1,62 @@
+##########################################################################################################
+# Items
+
+item.matter_scanner.name=MATTR SCANNR
+item.emergency_ration.name=EMERGENCY RASHUN
+item.phaser.name=FASR
+item.battery.name=BATTERY
+item.creative_battery.name=CREATIV BATTERY
+item.me_conversion_matrix.name=MATTR-ENERGY CONVERSHUN MATRIX
+item.isolinear_circuit.mk1.name=ISOLINEAR CIRCUIT MK1
```

Comment:
No new things for these?

Reply:
They are spelled slightly differently, and in all caps.

Reply:
Not this one: item.isolinear_circuit.mk1.name=ISOLINEAR CIRCUIT MK1